sway-lsp/src/capabilities/hover.rs

```diff
@@ -63,21 +63,28 @@ fn hover_format(token: &Token, ident: &Ident) -> Hover {
                 }
                 TypedDeclaration::FunctionDeclaration(func) => extract_fn_signature(&func.span()),
                 TypedDeclaration::StructDeclaration(decl_id) => {
-                    // TODO: do not use unwrap
-                    let struct_decl =
-                        declaration_engine::de_get_struct(decl_id.clone(), &decl.span()).unwrap();
-                    format_visibility_hover(struct_decl.visibility, decl.friendly_name())
+                    match declaration_engine::de_get_struct(decl_id.clone(), &decl.span()) {
+                        Ok(struct_decl) => {
+                            format_visibility_hover(struct_decl.visibility, decl.friendly_name())
+                        }
+                        Err(_) => token_name,
+                    }
```

Comment:
These could probably be `.map(|struct_decl| format_visibility_hover(...)).unwrap_or(token_name)` which is better than a `match`?  I don't know, it's subjective I think.

Reply:
That's a good suggestion, thanks! I'll make this change now.